homeassistant/components/overkiz/climate_entities/atlantic_pass_apc_zone_control_zone.py

```diff
@@ -0,0 +1,249 @@
+"""Support for Atlantic Pass APC Heating Control."""
+from __future__ import annotations
+
+from asyncio import sleep
+from typing import Any, cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.climate import PRESET_NONE, HVACMode
+from homeassistant.const import ATTR_TEMPERATURE
+
+from ..coordinator import OverkizDataUpdateCoordinator
+from .atlantic_pass_apc_heating_zone import AtlanticPassAPCHeatingZone
+from .atlantic_pass_apc_zone_control import OVERKIZ_TO_HVAC_MODE
+
+PRESET_SCHEDULE = "schedule"
+PRESET_MANUAL = "manual"
+
+OVERKIZ_MODE_TO_PRESET_MODES: dict[str, str] = {
+    OverkizCommandParam.MANU: PRESET_MANUAL,
+    OverkizCommandParam.INTERNAL_SCHEDULING: PRESET_SCHEDULE,
+}
+
+PRESET_MODES_TO_OVERKIZ = {v: k for k, v in OVERKIZ_MODE_TO_PRESET_MODES.items()}
+
+
+# Those device depends on a main probe that choose the operating mode (heating, cooling, ...)
+class AtlanticPassAPCZoneControlZone(AtlanticPassAPCHeatingZone):
+    """Representation of Atlantic Pass APC Heating And Cooling Zone Control."""
+
+    def __init__(
+        self, device_url: str, coordinator: OverkizDataUpdateCoordinator
+    ) -> None:
+        """Init method."""
+        super().__init__(device_url, coordinator)
+
+        # There is less supported functions, because they depend on the ZoneControl.
+        if not self.is_using_derogated_temperature_fallback:
+            # Modes are not configurable, they will follow current HVAC Mode of Zone Control.
+            self._attr_hvac_modes = []
+
+            # Those are available and tested presets on Shogun.
+            self._attr_preset_modes = [*PRESET_MODES_TO_OVERKIZ]
+
+        # Those APC Heating and Cooling probes depends on the zone control device (main probe).
+        # Only the base device (#1) can be used to get/set some states.
+        # Like to retrieve and set the current operating mode (heating, cooling, drying, off).
+        self.zone_control_device = self.executor.linked_device(1)
```

Comment:
Would be great if you can add this to a constant. See `somfy_thermostat.py`. This makes it a bit more clear where this number is coming from.